.github/workflows/notifications.json

```diff
@@ -0,0 +1,10 @@
+{
+  "include": [
+    {
+      "owner": "UKHomeOffice",
```

Comment:
Can we make this part of screts? It would better not to expose this in a open source repo code?

Reply:
It won't make a difference, these details are public since it is an open repo. you can query for them even if you are unauthenticated. They can't be triggered without a PAT which we are using with secrets